apps/examples/src/examples/text-search/README.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Text search
+component: ./TextSearchExample.tsx
+category: editor-api
+priority: 3
+keywords:
+  [
+    annotation,
```

Comment:
i think these keywords need updating maybe? the camera stuff doesn't seem related